sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/SortBasedAggregate.scala

```diff
@@ -27,6 +27,14 @@ import org.apache.spark.sql.execution.{UnsafeFixedWidthAggregationMap, SparkPlan
 import org.apache.spark.sql.execution.metric.SQLMetrics
 import org.apache.spark.sql.types.StructType
 
+/**
+ * Sort-based aggregation iterator.
+ *
+ * @param initialInputBufferOffset
```

Comment:
I think that this offset should be documented higher up, as part of InterpretedAggregateFunction. I'll clean this up while I add docs.